### ISSUE TITLE
Animate a marker

### ISSUE THREAD

**Issue by alvarouribe** [asennikov/ember-g-map#81]
I can't find the way to animate a marker...

**Comment by jeffrey008** [comment]
Hi there. I think there is no official way to animate a marker at the moment. But here is a workaround that you can try:

``` JavaScript
 {{#g-map-marker context lat=-30.041548 lng=-51.217107 title='Home' as |markerContext|}}
            {{animate-marker marker=markerContext}}
{{/g-map-marker}}
``` 

Then add the component `components/animate-marker.js`, inside init(), add
``` JavaScript
var marker = this.get('marker');
marker.setAnimation(google.maps.Animation.BOUNCE);  // Or whatever animation you like
```

I haven't try yet, but the basic idea is import the marker into component so we can add javscript to the marker. Good luck.

**Comment by alvarouribe** [comment]
I can access the marker but that didn't worked.
it says setAnimation is not a function so for some reason is not accessing the right element

**Comment by asennikov** [comment]
Try this:
```
{{#g-map-marker context lat=-30.041548 lng=-51.217107 title='Home' as |markerContext|}}
            {{animate-marker markerContext=markerContext}}
{{/g-map-marker}}
```

and then:
```
const marker = this.get('markerContext.marker');
marker.setAnimation(google.maps.Animation.BOUNCE);  // Or whatever animation you like
```

But clearly, we should just add official support for marker animation. However, let me know if that workaround works.

**Comment by marcemira** [comment]
:wave: I was just trying this out, but `markerContext.marker` it's undefined on `init()` (maps hasn't yet loaded at that point), so a `didInsertElement` would do better ;)

**Comment by steveszc** [comment]
Closing in favor of #96. Marker animations will be supported once support for MarkerOptions is done.
